Controller/Component/Cors/CorsComponent.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class CorsComponent extends Component {
+
+	public function initialize(Controller $controller) {
+		parent::initialize($controller); // TODO: Change the autogenerated stub
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1135)